test/fixtures/browser/umd-default-function-with-named-exports-module.js

```diff
@@ -0,0 +1,12 @@
+(function (root, factory) {
+  if (typeof define === 'function' && define.amd) {
+      define(['./amd-dep.js'], factory);
+  } else {
+    throw 'Test suit only supports amd'
```

Comment:
In this project, we usually use semicolons:

```suggestion
    throw 'Test suit only supports amd';
```